schedulingApp/unit-tests/test_accounts.py

```diff
@@ -1,25 +1,49 @@
 from django.test import TestCase
+from models import User
 
 
 # Fields: EMail Phone Address FirstName LastName PermissionLevel OfficeHours
 class CreateAccount(TestCase):
```

Comment:
Why are we testing internal code?  We should be testing some URL endpoint that creates accounts.